docs/storage.rst

```diff
@@ -0,0 +1,23 @@
+yogadl.storage
+==============
+
+.. autoclass:: yogadl.storage.LFSStorage
+   :noindex:
+   :members:
+
+.. autoclass:: yogadl.storage.LFSConfigurations
```

Comment:
Could we re-order these to have the configs come before the classes

Reply:
Sure.  I assume you still want them paired up?  Or all configs before any Storage classes?

Reply:
Yep the way you have it now is good.